### ISSUE TITLE
[A11y]Reset button for advanced search filtering does not have sufficient contrast

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8152]
![image](https://user-images.githubusercontent.com/11051729/89946316-94da2f80-dbd7-11ea-82c5-21c0a815589a.png)
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (#reset-advanced-search)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI)

Target application: NuGet Gallery | Packages - https://www.nuget.org/packages

Element path: #reset-advanced-search

Snippet: <input id="reset-advanced-search" class="btn btn-link form-control" type="button" value="Reset">

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.14 (foreground color: #337ab7, background color: #f4f4f4, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 84.0.4147.125

====

This accessibility issue was found using Accessibility Insights for Web 2.20.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
